python-sdk/pyproject.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
     "pyarrow",
     "python-frontmatter",
     "smart-open",
-    "SQLAlchemy>=1.3.18"
+    "SQLAlchemy>=1.3.18,<1.4.42"
```

Comment:
why we are restricting maybe a comment would be helpful

Reply:
We have it as part of the PR description, but no harm in adding in the code as well, will do.